tests/unit/gordo_components/test_gordo_model_info_provider.py

```diff
@@ -0,0 +1,23 @@
+from unittest.mock import Mock, patch
+
+import pytest
+
+from tests.unit.gordo_components.conftest import parametrized_get_model_data
+
+
+@pytest.mark.parametrize("data", parametrized_get_model_data)
```

Comment:
```suggestion
@pytest.mark.parametrize("projects", [["project-1", "project-2"], ["project-1", "missing_project", "project-2"]])
```
and update the tests accordingly.  Also get rid of `parametrized_get_model_data`.  Having a globally available object `parametrized_get_model_data` with lists, introduces a shared state which may lead to hard-to-debug issues if one of the tests alters a list.

Also instead of parametrization it seems it would be better to have 2 separate tests with models and side effects provided for each project explicitly.

Reply:
`parametrized_get_model_data` in the global scope make sense - moved to test.
But I don't see the reason of splitting the test on two separate, the code will be almost the same.